charts/akeyless-secure-remote-access/README.md

```diff
@@ -11,7 +11,7 @@ with signed certificate authentication, together with session recording.
 
 ## Introduction
 
-This chart bootstraps a Akeyless Zero Trust Bastion deployment,  Akeyless SSH bastion statefulset and a Akeyless Zero Trust Portal deployment on Kubernetes 
+This chart bootstraps a Akeyless Zero Trust Bastion deployment,  Akeyless SSH bastion statefulset and a Akeyless Zero Trust Portal deployment on Kubernetes
```

Comment:
It looks like we should not mention SSH bastion anymore, and just have ZTB (now statefulset) and ZTP, no?

Reply:
Not sure I undersatsnd, from what I understand from what we speak with @niv-ez, we want at first drop to make minimal changes to the chart- still have both services (ssh and ztb), just change the docker image (becauase now they use the same image), and in each service pass the relevant mode (I called this env variable `BASTION_TYPE`)
so ssh service start with `BASTION_TYPE=ssh-proxy` and web service start with `BASTION_TYPE=web`, but still these two services present in the chart.